src/clr-angular/data/datagrid/datagrid-footer.spec.ts

```diff
@@ -71,22 +50,22 @@ export default function(): void {
         clarityDirectiveSelection.selectionType = SelectionType.Multi;
         clarityDirectiveSelection.current.push(1);
 
-        context.clarityDirective.cdr.markForCheck();
+        //context.clarityDirective.cdr.markForCheck();
```

Comment:
Can we remove these?

Reply:
Yes, forgot to remove these ones. Done now, thanks 👌